src/quest_manager/models.py

```diff
@@ -975,3 +989,142 @@ def get_minutes_to_complete(self):
 
         minutes = (self.first_time_completed - self.timestamp).total_seconds() / 60
         return minutes
+
+
+class QuestionType(models.TextChoices):
+    """Identifies what types of questions can be created.
+
+    This is used in the views and forms to determine what type of question is being created, and to show/hide specific
+    fields. For example, with a short answer question, the solution_file field is hidden, but with a file upload question,
+    the solution_file field is shown.
+    """
+
+    SHORT_ANSWER = "short_answer", "Short Answer"
+    LONG_ANSWER = "long_answer", "Long Answer"
+    FILE_UPLOAD = "file_upload", "File Upload"
```

Comment:
Your human readable labels are identical to what would be inferred by, for example `FILE_UPLOAD.label`, so seems redundant and unnecessary.  See docs: https://docs.djangoproject.com/en/3.2/ref/models/fields/#enumeration-types

Reply:
You're right. You can do `question.get_type_display()` and it does the same thing.